lib/src/link/Link.tsx

```diff
@@ -16,9 +16,8 @@ const DxcLink = ({
   text = "",
   margin,
   tabIndex = 0,
-}) => {
+}: LinkCommonProps & (LinkTextProps | LinkIconProps)): JSX.Element => {
```

Comment:
Remove the intersection `LinkCommonProps &`. It is already done in each individual type `LinkTextProps` and `LinkIconProps`.

Reply:
Changed done.